code/game/gamemodes/clock_cult/clock_ratvar.dm

```diff
@@ -251,6 +255,9 @@
 			continue
 		N.clashing = TRUE
 		clash_of_the_titans(N) //IT'S TIME FOR THE BATTLE OF THE AGES
+		if(!admin_spawned)
+			for(var/mob/living/L in player_list)
```

Comment:
I feel like this needs some sort of limit.
A drone on the derelict or especially people playing ctf shouldn't be included.